graphstorm-processing/graphstorm_processing/data_transformations/dist_label_loader.py

```diff
@@ -55,6 +55,18 @@ def __post_init__(self) -> None:
             )
 
 
+@dataclass
+class CustomSplit:
+    """
+    Dataclass to hold the custom split for each of the train/val/test splits.
+    """
+
+    train: str
+    valid: str
+    test: str
+    column: list[str]
```

Comment:
Let's call this `mask_columns` or something else descriptive, the singular `column` does match the type (list)